### DISCUSSION TITLE
WakaTime Card Not Working

### DISCUSSION THREAD

**Discussion by ZacRayTho** [anuraghazra/github-readme-stats#2748] [Q&A] [answered]
Trying to add a wakatime card to my Readme. My wakatime account is public. When I add the card to my readme, it says there isnt a wakatime user. But if I go to the URL it shows me the card

https://github-readme-stats.vercel.app/api/wakatime?username=ZacRayTho

[![ZacRayTho's wakatime stats](https://github-readme-stats.vercel.app/api/wakatime?username=ZacRayTho)]

**Comment by qwerty541** [comment, marked as answer]
Currently wakatime card works fine in your profile readme. I think that your temporary problem can be connected with caching. See #1793
